sample-plugin/pom.xml

```diff
@@ -761,11 +761,18 @@
   </build>
   <profiles>
     <!-- Use .1 of most recent active line, and .3 (or later) of previous lines: https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#choosing-a-version -->
+    <profile>
+      <id>2.414.x</id>
+      <properties>
+        <bom>2.414.x</bom>
+        <jenkins.version>2.414</jenkins.version>
+      </properties>
+    </profile>
     <profile>
       <id>2.401.x</id>
       <properties>
         <bom>2.401.x</bom>
-        <jenkins.version>2.401.1</jenkins.version>
+        <jenkins.version>2.401.3</jenkins.version>
```

Comment:
I think according to the above logic this should remain 2.401.1 until 2.414.1 is released. At this point it does not much matter.